ui/src/table.tsx

```diff
@@ -584,7 +584,7 @@ const
           dataRows = (m.groups ? m.groups.flatMap(({ rows }) => rows) : m.rows)?.map(({ cells }) => cells) || [],
           data = toCSV([m.columns.map(({ label, name }) => label || name), ...dataRows]),
           a = document.createElement('a'),
-          blob = new Blob([data], { type: "octet/stream" }),
+          blob = new Blob(['\uFEFF', data], { type: "text/csv;charset=utf-8," }),
```

Comment:
Can you please also add a comment explaining why the prefix is necessary + link to the SO question for future reference? Thanks!